tokens/web/font.json5

```diff
@@ -1,8 +1,10 @@
 {
   font: {
       family: {
-        'serif': { value: '{options.font.family.serif-main.value}'},
-        'sans': { value: '{options.font.family.sans-main.value}'},
+        'serif-brand-font': { value: '{options.font.family.serif-main.value}'},
+        'serif': { value: '{options.font.family.serif.value}'},
```

Comment:
cdr-fonts uses these tokens (plain serif and sans) so if you are going to change them you'll have to update that as well or it will break. It only needs the single name, not the full stack.

Reply:
are they not self contained on font?
```
 'serif-main': { value: 'Stuart'},
 'serif-fallback': { value: 'Sentinel, Georgia, serif'},
```

Reply:
Or if you are referring to the cdr-fonts.css file on cedar -- yea I will update that one

Reply:
Specifically, there are 8 instances here that would need to be updated for the @font-face declarations. https://github.com/rei/rei-cedar/blob/next/src/css/fonts.scss

Reply:
correct but that file is not in this repo